Loop/Views/Presets/Components/RepeatOptionsView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  RepeatOptionsView.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 3/14/25.
+//  Copyright © 2025 LoopKit Authors. All rights reserved.
+//
+import SwiftUI
+
+struct RepeatOptionView: View {
+    let repeatOptions: PresetScheduleRepeatOptions
+
+    private var selectedDays: [PresetScheduleRepeatOptions] {
+        PresetScheduleRepeatOptions.allCases.filter { repeatOptions.contains($0) }
+    }
+
+    private var isSingleDay: Bool {
+        selectedDays.count == 1
+    }
+
+    var body: some View {
+        if repeatOptions == .none {
+            Text(repeatOptions.description)
+                .tint(.secondary)
+        } else if isSingleDay {
+            Text(selectedDays[0].description)
+                .foregroundColor(.secondary)
+        } else {
+            HStack(spacing: 4) {
+                ForEach(PresetScheduleRepeatOptions.allCases, id: \.rawValue) { day in
+                    Text(String(day.description.first!))
+                        .font(.system(size: 12))
```

Comment:
Not sure if we want this to scale with dynamic type If so you could add a property to the view:
```swift
@ScaledMetric var dayTextSize: Double = 12
```
and then use that for the size:
```swift
.font(.system(size: dayTextSize))
```